src/System.Text.Json/src/System/Text/Json/Utf8JsonReader.cs

```diff
@@ -390,7 +340,7 @@ private bool ReadFirstToken(byte first)
             if (first == JsonConstants.OpenBrace)
             {
                 _currentDepth++;
-                _stackFreeContainer = 1;
+                _bitStack._allocationFreeContainer = 1;
```

Comment:
Should I keep `_allocationFreeContainer` private and just use the PushTrueAt public API instead? I wanted to avoid the depth check and bit-wise operations.

Reply:
Either that or have a special method like `SetToOneBit()`